pkg/controller/account/account_controller.go

```diff
@@ -536,9 +548,35 @@ func (r *ReconcileAccount) nonCCSAssignAccountID(reqLogger logr.Logger, currentA
 	// update account cr with awsAccountID from aws
 	currentAcctInstance.Spec.AwsAccountID = awsAccountID
 
+	// tag account with hive shard name
+	err = r.tagAccount(reqLogger, awsSetupClient, awsAccountID)
+	if err != nil {
+		reqLogger.Info("Unable to tag aws account.", "account", currentAcctInstance.Name, "AWSAccountID", awsAccountID, "Error", error.Error(err))
+	}
+
 	return r.Client.Update(context.TODO(), currentAcctInstance)
 }
 
+func (r *ReconcileAccount) tagAccount(reqLogger logr.Logger, awsSetupClient awsclient.Client, awsAccountID string) error {
+
+	inputTag := &organizations.TagResourceInput{
+		ResourceId: aws.String(awsAccountID),
+		Tags: []*organizations.Tag{
+			{
+				Key:   aws.String("owner"),
+				Value: aws.String(r.shardName),
+			},
+		},
+	}
+
+	_, err := awsSetupClient.TagResource(inputTag)
+	if err != nil {
```

Comment:
Here you can just return err directly I'm guessing? As it should return nil if it's not an err, so no need to check

Reply:
it's telling me i need 2 variables so i'm the "_" in there